engine_test.go

```diff
@@ -226,6 +226,14 @@ var queries = []struct {
 			{"     └─ Column(s, TEXT, nullable=false)"},
 		},
 	},
+	{
+		`SELECT split(s," ") FROM mytable`,
```

Comment:
can you split by, per example, `s`?

Reply:
I already added another test splitting by 's'